connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -91,6 +91,16 @@ message WriteOperation {
   // (Optional) A list of configuration options.
   map<string, string> options = 9;
 
+  message SaveTable {
+    string table_name = 1;
+    TableSaveMethod save_method = 2;
+
+    enum TableSaveMethod {
+      SAVE_AS_TABLE = 0;
+      INSERT_INTO = 1;
```

Comment:
what about adding `SAVE_MODE_ INSERT_INTO` into existing `enum SaveMode`?

Reply:
I'm afraid I don't think it's a good idea. The method and mode are orthogonal concepts. Spark Connect should let Spark decide how it should behave with the options.

Reply:
Got it, Thanks for the clarification!